StreamingAnalytics/README.md

```diff
@@ -22,13 +22,13 @@ modify or extend the examples or implement a real solution with Apama and
 thin-edge.io we recommend you follow the guidance on creating, configuring and
 deploying a project contained within this document.
 
-- A full installation of [Apama Community Edition](https://apamacommunity.com/downloads/)
-including Software AG Designer on a laptop or PC.
+- The full development package of [Apama Community Edition](https://apamacommunity.com/)
+(not the core package, which does not include Eclipse).
```

Comment:
The full development package of Apama Community Edition. (Unlike the core package, this includes Eclipse.)